src/pages/careers/index.js

```diff
@@ -10,7 +10,7 @@ import Layout from 'components/layout/layout'
 import { localize, WithIntl } from 'components/localization'
 
 const Careers = () => (
-    <Layout type="careers" padding_top="7rem">
+    <Layout type="careers" margin_top="7rem">
```

Comment:
The issue seems to have been fixed 👍 . Just wanted some clarification, when do you use `margin top` and when do you use `top`?

Reply:
`margin-top` is when you want to make a space between the selected container and the one above it. It won't affect the other containers since all of them will be shifted with the same amount of pixels. `top` is to shift the container by amount of pixels but it will affect the other containers.

Reply:
to be more clearer see the [deployment of `top`](https://deriv-com-5b96mb25u.binary.sx/) and [deployment of `margin-top`](https://deriv-com-h021vpwou.binary.sx/) notice that the logo in footer is covered by the element above it. where it doesn't occur in the latter.